build-utils.js

```diff
@@ -94,41 +97,85 @@ function taskHarness(task, projectOrStar, ...args) {
  * also add a license header and create sourcemaps.
  *
  * @param  {Object} options Options object with 'projectDir' and 'rollupConfig'
- * @return {[type]}         [description]
+ * @return {Promise}
  */
 function buildJSBundle(options) {
-  const destPath = path.join(options.projectDir, 'build');
-  const licenseHeader = fs.readFileSync('LICENSE-HEADER', 'utf8');
+  return new Promise((resolve, reject) => {
+    const outputName = options.buildPath.replace(/^build\//, '');
+    const sources = [
+      `${options.projectDir}/src/**/*.js`,
+      '{lib,packages}/**/*.js',
+      'node_modules/*/**/*.js',
```

Comment:
Maybe? I think you were the one who used that glob pattern to begin with: https://github.com/GoogleChrome/sw-helpers/blame/master/build-utils.js#L112 and I was just keeping it consistent.

I'm happy to change it and see if there's an issue.